### PR TITLE
[O2B-692] Add run definition column and filtering in runs overview

### PR DESCRIPTION
**Pull request by martinboulais**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [x] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,12 @@ All notable changes to this project will be documented in this file. See [standa
 * Notable changes for users:
   * `pdpWorkflowParameters, pdpBeamType, readoutCfgUri` field added to the runs detail page and can be exported.
   * Time after last run and corresponding loss has been added to fill statistics
+  * `definition` is now available in un overview's table and filtering
```
Comment (JikHuijberts) [inline CHANGELOG.md:9]: typo

Comment (martinboulais) [reply CHANGELOG.md:9]: fixed

File: lib/database/adapters/RunAdapter.js [hunk 2]
```diff
@@ -79,12 +80,13 @@ class RunAdapter {
...
+            runType,
+            definition: getRunDefinition(databaseObject),
+            runQuality,
+            nDetectors,
+            nFlps,
+            nEpns,
+            dd_flp,
             dcs: dcs,
             epn: epn,
             epnTopology: epnTopology,
```
[the remark this answers is not in the document]
Comment (martinboulais) [reply lib/database/adapters/RunAdapter.js:92]: My IDE colored them in another color 🙄 
Fixed

File: lib/database/utilities/QueryBuilder.js [hunk 3]
```diff
@@ -428,20 +428,21 @@ module.exports = (sequelize) => {
         }
 
         /**
-         * Adds a filter on the given column.
+         * Adds a filter on the given subject.
          *
-         * @param {String} column The target column.
+         * @param {string|object} subject The target subject.
          * @returns {WhereQueryBuilder} The WhereQueryBuilder instance.
```
[the remark this answers is not in the document]
Comment (martinboulais) [reply lib/database/utilities/QueryBuilder.js:434]: Nope, fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [0.38.0](https://github.com/AliceO2Group/Bookkeeping/releases/tag/%40aliceo2%2Fbookkeeping%400.38.0)
```
Comment (JikHuijberts) [inline CHANGELOG.md:5]: should be `0.39.0`

**Review by JikHuijberts — DISMISSED** [review]

**Review by graduta — APPROVED** [review]

**Review by graduta — APPROVED** [review]

**Pull request merged**